sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -312,24 +312,59 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * @param value The string value of the instance.
      */
     explicit KeyEncryptionAlgorithm(std::string value) : ExtendableEnumeration(std::move(value)) {}
-
+#ifndef CKM_RSA_AES_KEY_WRAP
     /**
      * @brief Gets the CKM_RSA_AES_KEY_WRAP algorithm.
      *
+     * @deprecated Use KeyEncryptionAlgorithm::CkmRsaAesKeyWrap.
      */
-    AZ_SECURITY_KEYVAULT_KEYS_DLLEXPORT static const KeyEncryptionAlgorithm CKM_RSA_AES_KEY_WRAP;
+    [[deprecated("Use "
```

Comment:
I'd recommend adding a changelog entry to inform customers of this extensible enum rename.